dpbench/configs/framework_info/numba_mlir.toml

```diff
@@ -0,0 +1,10 @@
+# SPDX-FileCopyrightText: 2022 - 2023 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+
+[framework]
+simple_name = "numba_mlir"
+full_name = "numba_mlir"
+prefix = "mlir"
+class = "NumbaMlirFramework"
+sycl_device = "opencl:cpu:0"
```

Comment:
I'm out of context here, but does `numba_mlir` runs on top of the sycl?

Reply:
We are using sycl to run gpu code